### PR TITLE
Owens v18.1 test

### PR DESCRIPTION
**Pull request by brianmcmichael**
See INC0324321

Updated the application to launch Ansys v18.1 on Owens cluster.

App launches successfully on standard and GPU node. Unable to test hugemem due to queue time.

This should be tested with an example job that utilizes the underlying solver(s).

### CODE REVIEW

File: form.yml [hunk 1]
```diff
@@ -17,24 +17,21 @@ attributes:
...
+      - **any** - (*28 cores*) Use any available Owens node. This reduces the
         wait time as there are no node requirements.
-      - **vis** - (*12 cores*) Use an Oakley node that has an NVIDIA Tesla
-        M2070 GPU with hardware rendering enabled for 3D visualization. There
-        are 128 of these nodes on Oakley.
-      - **bigmem** - (*12 cores*) Use an Oakley node that has 192GB of
-        available RAM. There are 8 of these nodes on Oakley.
-      - **hugemem** - (*32 cores*) Use an Oakley node that has 1TB of available
-        RAM as well as 32 cores. There are 1 of these nodes on Oakley.
+      - **gpu** - (*28 cores*) Use an Owens node that has an NVIDIA P100 "Pascal" GPU with hardware rendering enabled for 3D visualization. There are 160 of these nodes on Owens.
```
Comment (nickjer) [inline form.yml]: Why did you remove `:vis` support? Does it successfully launch VirtualGL without it on Owens?

File: form.yml [hunk 2]
```diff
@@ -17,24 +17,21 @@ attributes:
...
     options:
-      - [ "any",     ":ppn=12"            ]
-      - [ "vis",     ":ppn=12:vis:gpus=1" ]
-      - [ "bigmem",  ":ppn=12:bigmem"     ]
-      - [ "hugemem", ":ppn=32:hugemem"    ]
+      - [ "any",     ":ppn=28"            ]
+      - [ "gpu",     ":ppn=28:gpus=1"     ]
+      - [ "hugemem", ":ppn=48:hugemem"    ]
   version:
     widget: select
```
Comment (nickjer) [inline form.yml]: If there is a single solver version on Owens, it may not make sense to a use a dropdown select field.

**Comment by nickjer** [comment]
Have you been able to test some of the examples for Fluent and CFX with MPI that are usually included in the installation directory? We will need to test MPI support before deploying this to Owens.

File: form.yml (hunk 1 above)
Comment (nickjer) [reply form.yml]: You can also use the same help text we use for the Abaqus app:

https://github.com/OSC/bc_osc_abaqus/blob/27053997ef29d438349787acd3d0ea0d9ead9ce7/form.yml#L18-L21

File: form.yml [hunk 3]
```diff
@@ -17,24 +17,21 @@ attributes:
...
-    label: "ANSYS Workbench version"
+    label: "Ansys Workbench version"
```
Comment (nickjer) [inline form.yml]: All of the ANSYS marketing material uses all-caps.

image

I'd keep it in line with their branding style.

File: form.yml (hunk 1 above)
Comment (brianmcmichael) [reply form.yml]: There are some fixes that have been added in the https://github.com/OSC/bc_osc_ansys_workbench/tree/owens18_PMIU0149 branch. Those will need to be backported to here.

File: form.yml [hunk 4]
```diff
@@ -17,24 +17,24 @@ attributes:
     widget: select
     label: "Node type"
     help: |
-      - **any** - (*12 cores*) Use any available Oakley node. This reduces the
+      - **vis** - (*28 cores*) This node includes an NVIDIA Tesla P100 GPU
```
Comment (nickjer) [inline form.yml]: Do we want `vis` node types to be the default? They will have to wait longer in queue.

Comment (ericfranz) [reply form.yml]: That was a requirement for the class. Not a requirement for the port for everyone to use. Now we have a separate class specific branch.

Comment (brianmcmichael) [reply form.yml]: Ok, I'll bump it back.

File: form.yml [hunk 5]
```diff
@@ -17,24 +17,24 @@ attributes:
...
+      - [ "vis",     ":ppn=28:vis:gpus=1" ]
+      - [ "hugemem", ":ppn=48:hugemem"    ]
   version:
     widget: select
     label: "ANSYS Workbench version"
     help: "This defines the version of ANSYS you want to load."
     options:
-      - [ "15.0.7", "ansys/15.0.7" ]
-      - [ "18.1",   "ansys/18.1"   ]
+      - [ "18.1", "ansys/.18.1" ]
```
Comment (nickjer) [inline form.yml]: Just mentioning again since my previous comment was removed by GitHub, but since there is only one option a `<select>` input is not necessary.

Comment (ericfranz) [reply form.yml]: Yes. But there will be more eventually. In the branch for the class we got rid of the dropdown.

Comment (brianmcmichael) [reply form.yml]: This PR is not ready for primetime anyways. The `ansys/.18.1` module is a hidden module (experimental, I think) that we shouldn't be using in production. This will need to be updated with the correct modules(s) once we know what those will be.

Comment (brianmcmichael) [reply form.yml]: > that we shouldn't be using in production

With the exception of the upcoming course.

**Comment by brianmcmichael** [comment]
untitled

File: manifest.yml [hunk 6]
```diff
@@ -1,13 +1,13 @@
 ---
-name: ANSYS Workbench
+name: ANSYS Workbench (Owens)
```
Comment (ericfranz) [inline manifest.yml]: Since we are just replacing the existing ANSYS installation, keep the title the same as before, so remove `(Owens)`

File: manifest.yml [hunk 7]
```diff
@@ -1,13 +1,13 @@
...
 description: |
   This app will launch an [ANSYS Workbench](http://www.ansys.com/) within a
   [Turbo VNC](https://www.turbovnc.org/) server on the
-  [Oakley cluster](https://www.osc.edu/resources/technical_support/supercomputers/oakley).
-  
+  [Owens cluster](https://www.osc.edu/resources/technical_support/supercomputers/owens).
+
   **Warning: 15.0.7 will stop being available in March when this app is updated to submit jobs to Owens.
   Please transition to using a later version of ANSYS Workbench. If the departure of 15.0.7
   will be problematic, please contact oschelp@osc.edu**
```
Comment (ericfranz) [inline manifest.yml]: Remove this warning since you removed 15.0.7 as an option.

**Comment by ericfranz** [comment]
Added two small comments. Otherwise everything looks good as it appears you merged changes previously.

**Comment by ericfranz** [comment]
Might want to manually deploy and test on ondemand-dev prior to making a release and building an rpm

**Comment by nickjer** [comment]
I merged these changes into `master` individually as GitHub wasn't showing the correct changes here.
